x-pack/plugins/synthetics/server/synthetics_service/project_monitor_formatter.ts

```diff
@@ -266,24 +266,24 @@ export class ProjectMonitorFormatter {
       attributes: { [ConfigKey.REVISION]: _, ...normalizedPreviousMonitorAttributes },
     } = normalizeSecrets(decryptedPreviousMonitor);
     const hasMonitorBeenEdited = !isEqual(normalizedMonitor, normalizedPreviousMonitorAttributes);
-    const monitorWithRevision = formatSecrets({
-      ...normalizedPreviousMonitorAttributes, // ensures monitor AAD remains consistent in the event of field name changes
-      ...normalizedMonitor,
-      revision: hasMonitorBeenEdited
-        ? (previousMonitor.attributes[ConfigKey.REVISION] || 0) + 1
-        : previousMonitor.attributes[ConfigKey.REVISION],
-    });
-    const editedMonitor: SavedObjectsUpdateResponse<EncryptedSyntheticsMonitor> =
-      await this.savedObjectsClient.update<MonitorFields>(
-        syntheticsMonitorType,
-        previousMonitor.id,
-        {
-          ...monitorWithRevision,
-          urls: '',
-        }
-      );
 
     if (hasMonitorBeenEdited) {
+      const monitorWithRevision = formatSecrets({
```

Comment:
we should also skip, fleet operations no?

Reply:
Discussed offline, we already do.